.github/workflows/build-admin-ui.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
             node-modules-
       - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
Should we try 20?

Reply:
We could try that but I was keeping this in line with what [ember suggest](https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md) for `v4.12.2`. I know we still plan to get to ember v5 and that would push us into node v20.
This also accurately reflects which version we use in [boundary-ui-releases workflow](https://github.com/hashicorp/boundary-ui-releases/blob/main/.github/workflows/build-packages.yml).

Reply:
To argue against my own statement, we could test updating our workflows to node v20. Then we would have one less hurdle to getting ember to v5.

Reply:
Yup, I was going to suggest we should be trying to push our CI and workflows to 20 since we'd still be behind if we just go to 18.

Reply:
Indeed in this comment, 20 should be the one we aim for. Nice work!!